tests/integration/test_searchsorted.py

```diff
@@ -176,6 +176,52 @@ def test_ndim(ndim, side):
     check_api(a, None, v, side)
 
 
+def test_ndim_positive():
```

Comment:
The name of this test is very confusing. I would suggest collecting all the error-checking tests into a class `Text_searchsorted_Errors` (or whatever, similar to other recent PRs) and then this coud be `test_ndim_mismatch` or something  like that.

Reply:
fix